File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/ShareVerifyVO.java
package com.sdkwork.app.model;

public class ShareVerifyVO {
    private String createdAt;
    private String updatedAt;
    private Boolean valid;
    private String errorMessage;
    private ShareInfoVO shareInfo;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public Boolean getValid() {
        return this.valid;
    }
    
    public void setValid(Boolean valid) {
        this.valid = valid;
    }

    public String getErrorMessage() {
        return this.errorMessage;
    }
    
    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }

    public ShareInfoVO getShareInfo() {
        return this.shareInfo;
    }
    
    public void setShareInfo(ShareInfoVO shareInfo) {
        this.shareInfo = shareInfo;
    }
}

File: sdkwork-app-sdk-typescript/src/types/share-verify-vo.ts
import type { ShareInfoVO } from './share-info-vo';

/** 分享验证响应 */
export interface ShareVerifyVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 验证结果 */
  valid?: boolean;
  /** 错误信息 */
  errorMessage?: string;
  /** 分享信息 */
  shareInfo?: ShareInfoVO;
}
